cypress/e2e/api/engage-sphere-customers.spec.cy.js

```diff
@@ -0,0 +1,243 @@
+const CUSTOMERS_API_URL = `${Cypress.env('API_URL')}/customers`
+describe('API - Get Customers', () => {
+    context('Valid requests', () => {
+        it('GET /customers - returns 200 and valid response structure (default parameters)', () => {
+            cy.request({
+                method: 'GET',
+                url: CUSTOMERS_API_URL,
+            }).then(({ status, body }) => {
+                expect(status).to.eq(200);
+                expect(body).to.have.property('customers').and.be.an('array').and.not.be.empty;
+                body.customers.forEach((customer) => {
+                    expect(customer).to.have.all.keys(
+                        'id',
+                        'name',
+                        'employees',
+                        'size',
+                        'industry',
+                        'contactInfo',
+                        'address'
+                    );
+                    expect(customer.id).to.be.a('number');
+                    expect(customer.name).to.be.a('string');
+                    expect(customer.employees).to.be.a('number');
+                    expect(customer.size).to.be.a('string');
+                    expect(customer.industry).to.be.a('string');
+                    if (customer.contactInfo !== null) {
```

Comment:
Uma versão simplificada seria a seguinte:

```suggestion
                    if (customer.contactInfo) {
```